fe/fe-core/src/main/java/com/starrocks/http/rest/v2/vo/TablePartitionView.java

```diff
@@ -0,0 +1,246 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.http.rest.v2.vo;
```

Comment:
what does this `vo` stands for?

Reply:
@kevincai `VO` is the abbreviation of View Object, which is used to organize and encapsulate the persistence layer data in the desired form and return it to the client, while shielding the details of the persistence layer data (such as entity object). The [Oracle Docs](https://docs.oracle.com/cd/A97688_16/generic.903/bc_guide/bc_awhatisavo.htm) give more introduction.

Reply:
Could you just make the package name `xxx.viewobj`? because i think without reading the above explanation, most of the new developers cannot understand the abbreviation from the code directly.

Reply:
Just a thought, maybe a better solution is making all the concret view object class extends from an abstract class `ViewObject`(even an empty class is acceptable), which will make the code better arranged and you also have a place to give your explanation about the design.

Reply:
@nshangyiming I have add a `package-info.java` file to describe it in the `xxx.vo` package.